### PR TITLE
Grammatical and spelling corrections

### PR DESCRIPTION
**Pull request by DavidLiedle**
Minor, text-only changes to improve the README copy.

### CODE REVIEW

**Comment by andrerom** [comment]
+1

thanks, several off this week, but for the second +1 maybe @pspanja can have a look :)

side: the Readme is overdue for a major update on several repos, to make info consistent, and to concretize the info as there is too much all over the place here now for instance. But that is more something we should hack together in doc first and look at across all.

File: Readme.md [hunk 1]
```diff
@@ -1,123 +1,120 @@
...
 - [eZ/Publish/SPI](eZ/Publish/SPI/)  SPI's are *Service Provider Interfaces*, *not yet frozen*.
-- [eZ/Publish/Core](eZ/Publish/Core/) is where you'll find implementations of both API and SPI's; the naming aims to map to name of interface they implement, example `Core\Persistence\Legacy` being implementation of `SPI\Persistence`.
+- [eZ/Publish/Core](eZ/Publish/Core/) is where you'll find implementations of both APIs and SPIs; the naming aims to map to name of the interface they implement. For example, `Core\Persistence\Legacy` being implementation of `SPI\Persistence`.
 
 
-### Testing locally
+### Testing Locally
 
-This kernel contains a comrehensive set of unit, functional and integration tests, at time of writing 9k unit tests, 5.6k integration tests and several functional tests.
+This kernel contains a comprehensive set of unit, functional, and integration tests. At the time of writing, 9k unit tests, 5.6k integration tests, and several functional tests.
```
Comment (pspanja) [inline Readme.md]: 8.9k integration tests :)

**Comment by pspanja** [comment]
Looks good to me, +1 aside from one remark above.

**Comment by andrerom** [comment]
I'll fix the remark after merge :)

**Comment by DavidLiedle** [comment]
Thanks guys!

**Pull request merged**